tests/php/sync/test_class.jetpack-sync-themes.php

```diff
@@ -34,6 +34,65 @@ function theme_info() {
 class WP_Test_Jetpack_Sync_Themes extends WP_Test_Jetpack_Sync_Base {
 	protected $theme;
 
+	/**
+	 * Dummy Themes.
+	 *
+	 * @var string[]
+	 */
+	protected $themes = array(
+		'theme-file-sync-parent',
+		'theme-file-sync-child',
+	);
+
+	/**
+	 * Move Dummy Themes to proper location for testing.
+	 */
+	public static function setUpBeforeClass() {
+		parent::setUpBeforeClass();
+
+		$themes = array(
+			'theme-file-sync-parent',
+			'theme-file-sync-child',
+		);
+
+		// Copy themes from tests/php/files/ to wp-content/themes.
+		foreach ( $themes as $theme ) {
+			$source_dir = __DIR__ . '/../files/' . $theme;
+			$dest_dir   = WP_CONTENT_DIR . '/themes/' . $theme;
+
+			mkdir( $dest_dir );
+
+			foreach ( glob( $source_dir . '/*.*' ) as $theme_file ) {
+				copy( $theme_file, $dest_dir . '/' . basename( $theme_file ) );
+			}
+		}
+
+	}
+
+	/**
+	 * Remove Dummy Themes.
+	 */
+	public static function tearDownAfterClass() {
+		parent::tearDownAfterClass();
+
+		$themes = array(
+			'theme-file-sync-parent',
+			'theme-file-sync-child',
+		);
+
+		// Copy themes from tests/php/files/ to wp-content/themes.
+		foreach ( $themes as $theme ) {
+			$dest_dir = WP_CONTENT_DIR . '/themes/' . $theme;
+
+			foreach ( glob( $dest_dir . '/*.*' ) as $theme_file ) {
+				unlink( $theme_file );
+			}
+
+			rmdir( $dest_dir );
+		}
+
+	}
+
 	public function setUp() {
 		parent::setUp();
 		$themes      = array( 'twentyten', 'twentyeleven', 'twentytwelve', 'twentythirteen', 'twentyfourteen' );
```

Comment:
What's going on here? Can we remove the random switching to a theme that doesn't exist in the installation anymore?

Locally if I just set `$this->theme = 'doesnotexist';` the tests still seem to pass.

Reply:
removed in latest update.